compiler/rustc_ast_pretty/src/pprust/state/expr.rs

```diff
@@ -488,7 +488,18 @@ impl<'a> State<'a> {
                         self.space();
                     }
                     MatchKind::Postfix => {
+                        let need_par = matches!(expr.kind, ast::ExprKind::Binary(_, _, _));
```

Comment:
I looked at the [Expression precedence][1] table of the Reference and found nothing else to wrap in parentheses.

Please comment if I missed something!

[1]: https://doc.rust-lang.org/reference/expressions.html#expression-precedence

Reply:
It's probably best to use https://doc.rust-lang.org/nightly/nightly-rustc/rustc_ast/ast/struct.Expr.html#method.precedence instead of matching exhaustively on `ExprKind`

like `.precedence() <= PREC_SOMETHING` or `<` or something (idk which predecence to choose)

Reply:
Thanks for your comment. This allowed me to fix a few other instances of the bug reported by #124206!